inscriptions/ISic020431.xml

```diff
@@ -156,7 +156,7 @@
             <listChange>
                 <change when="2020-10-05" who="#SS">Simona Stoyanova normalised Unicode</change>
                 <change when="2020-10-08" who="#SS">Simona Stoyanova updated list of languages</change>
-                <change when="2021-02-16" who="#VM">Valentina Mignosa added metadata, text</change>
+                <change when="2022-03-31" who="#VM">Valentina Mignosa added metadata, text</change>
```

Comment:
i've been wondering about this deletion of the change element - you're adding / extending the metadata and text; but you did create something on the previous date, so I'm not sure why one deletes the change note (that seems to me to be poor practice and to miss the point of change notes)? At most, amend the previous change description to something you now think more accurate, such as 'created preliminary blank file' or similar?